### PR TITLE
Raise cost for purchasing new cashcards to 10$.

### LINKED ISSUE #33850: (Nearly) Infinite Money Exploit
# Describe the bug

Cash cards have the same value regardless of the amount of cash in them. This value is always more than the cost to purchase a new cash card from an ATM($1.00). You can buy large amounts of cards from ATMs and trade them for items worth more than 4 to 8 times what you spend to get the cards(Depends on barter level).

# Steps To Reproduce

1. Find an ATM.
2. Buy as many cards as you can with your starting bank funds.
3. Trade them to a vendor for more than you paid for them.

# Expected behavior

Cash cards should be worth less than or equal to what you pay for them, or be worth however much you have loaded on the card.

# Screenshots

image

image

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.D-7350-gf619385 [64-bit]
 - Graphics version: Tiles
 - Mods loaded: Vanilla

# Additional context

This is my first bug report, wish you all the best!

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Bugfixes "Raise cost for purchasing new cashcards to 10$."```
#### Purpose of change

Close #33850
#### Describe the solution

Exactly what it says in title and summary.
#### Describe alternatives you've considered

Raise it even more, if it wasn't enough.

### CODE REVIEW

**Comment by curstwist** [comment]
making a pre-cataclysm replacement card cost 100$ doesn't make much sense.  Maybe their post-apocalypse price should be adjusted down instead.

I'd be ok for them costs 5-10$ to replace though, banks love their fees.

**Comment by arijust** [comment]
It seems postapoc_price of cash cards was already reduced to 1 cent.
Should I just close this PR, instead?

File: data/json/items/generic.json [hunk 1]
```diff
@@ -2408,7 +2408,7 @@
     "name": "cash card",
     "description": "A yellow plastic card used to store money.  These became popular once the government officially moved to all electronic money.  It holds up to 2 million dollars.",
     "price": 100,
```
Comment (Qrox) [inline data/json/items/generic.json]:
```suggestion
    "price": 100,
```
Should this also be changed to 1000?

**Comment by I-am-Erk** [comment]
Give it a test, see if you can still exploit them at this value. I think you'll be safe

**Comment by arijust** [comment]
Alright, it seems to be fine now.

**Pull request merged**